src/rockstor/storageadmin/static/storageadmin/js/templates/common/navbar.jst

```diff
@@ -40,6 +40,7 @@
       <li><a href="#config-backup"><i class="glyphicon glyphicon-floppy-save"></i> Config Backups</a></li>
       <li><a href="#email"><i class="fa fa-envelope"></i> Email Alerts</a></li>
       <li><a href="#logsmanager"><i class="fa fa-gears"></i> Logs Manager</a></li>
+      <li><a href="#shell"><i class="fa fa-terminal"></i> System Shell</a></li>
```

Comment:
I agree that System navbar item is the right place for this. But what do you think of a small terminal icon right next to System Time? It looks nicer IMO, just an idea.